tests/unit/tree/test_gdrive.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from dvc.exceptions import DvcException
+from dvc.tree import GDriveTree
+from dvc.tree.gdrive import GDriveURLInfo
+from tests.basic_env import TestDvc
+from tests.utils import load_gdrive_credentials
+
+
+class TestGDriveURL(TestDvc):
+    def setUp(self):
+        super().setUp()
+        success = load_gdrive_credentials(self.dvc.root_dir)
+        if not success:
+            pytest.skip("no gdrive-user-credentials.json available")
```

Comment:
We'll need to run this on CI. I think reading creds from the file is not a very regular approach for that. We already should have some tests (they might be located in the remotes). I think we read credentials from the env var. Could you please check? Does it make sense to use the same approach here?

Reply:
Definitively, playing with the existing mechanism is a better approach. I'll address it.

Reply:
Thanks 🙏

Reply:
I've changed to rely on the env var. When I ran the tests with the env var configured, the code executed some tests cases that had being skipped. One of them is `test_data_cloud.test_cloud_cli`, which is now failing. I checked, and it fails when trying to upload an empty file (which raises an exception because Google Drive server returns a 400 error code). I'm wondering if this error is associated to the changes I didn't, or any kind of misconfiguration in either my Google Drive account or its credentials.

Reply:
I've just run the tests in the `master` branch and I got the same error in `test_data_cloud.test_cloud_cli`, so I'm assuming it's a misconfiguration in my environment.